### DISCUSSION TITLE
ActionGroup component errors out

### DISCUSSION THREAD

**Discussion by conormdowney** [adobe/react-spectrum#2115] [Q&A] [answered]
So Im just trying to get a simple ActionGroup (https://react-spectrum.adobe.com/react-spectrum/ActionGroup.html) with overflow enabled to work.

So I added the following code to my component: 

`const menu = (<ActionGroup overflowMode="collapse" maxWidth={250}>
    <Item key="edit">
      <Draw />
      <Text>Edit</Text>
    </Item>
    <Item key="copy">
      <Copy />
      <Text>Copy</Text>
    </Item>
    <Item key="delete">
      <Delete />
      <Text>Delete</Text>
    </Item>
    <Item key="move">
      <Move />
      <Text>Move</Text>
    </Item>
    <Item key="duplicate">
      <Duplicate />
      <Text>Duplicate</Text>
    </Item>
  </ActionGroup>)`

Then i added menu to my jsx. But it is giving me an error when it tries to render. The error happens in the actiongroup code. it is expecting there to be a theme but there its not finding one. The coe is in 

`node_modules/@react-spectrum/actiongroup/node_modules/@react-spectrum/provider/dist/main.js`

Now that im typing it out i dont know why thereis a node modules inside the actiogroup node module....??
I used the install command from here: https://www.npmjs.com/package/@react-spectrum/actiongroup
`npm i @react-spectrum/actiongroup`

**Comment by snowystinger** [comment, marked as answer]
It's best to install `@adobe/react-spectrum`
What you're missing is the React Spectrum Provider at the top of you App, see https://react-spectrum.adobe.com/react-spectrum/Provider.html
As for why it has a node_modules, you only installed the one package, and maybe didn't top level the dependencies. This can happen if you already have some other react spectrum components installed that have a different version of Provider.
If you install the whole thing, then this should become a moot point.
